featurebyte/session/databricks_unity.py

```diff
@@ -15,16 +15,17 @@ class DatabricksUnitySchemaInitializer(BaseSparkSchemaInitializer):
     Databricks unity schema initializer
     """
 
-    async def initialize(self) -> None:
-        await super().initialize()
+    @property
+    def current_working_schema_version(self) -> int:
+        return 13
+
+    async def create_schema(self) -> None:
+        await super().create_schema()
+        # grant permissions on schema to the group
```

Comment:
move this to only run when schema is created instead of on every session creation

Reply:
Please include this change in the PR title before merging.